src/server/game/Loot/LootMgr.cpp

```diff
@@ -658,28 +658,36 @@ QuestItemList* Loot::FillQuestLoot(Player* player)
     if (items.size() == MAX_NR_LOOT_ITEMS)
         return nullptr;
 
-    QuestItemList* ql = new QuestItemList();
+    auto* ql = new QuestItemList();
```

Comment:
I'd prefer to refrain to use auto in this case, to easier know what this variable is pointing to.